docs/getting_started/tutorial.md

```diff
@@ -158,16 +162,18 @@ it means that this field is not required and can be empty. You
 can use any of the [Django core fields](https://docs.djangoproject.com/en/stable/ref/models/fields). `content_panels` define the
 capabilities and the layout of the editing interface. When you add fields to `content_panels`, it enables them to be edited on the Wagtail interface. [More on creating Page models](../topics/pages).
 
-Run `python manage.py makemigrations` (this will create the migrations file), then
-`python manage.py migrate` (this executes the migrations and updates the database with your model
-changes). You must run the above commands each time you make changes to
-the model definition.
+Run:
+
+```console
+$ python manage.py makemigrations # creates the migrations file
+$ python manage.py migrate # executes the migrations and updates the database with your model changes 
```

Comment:
Nice! Let’s rework this a bit further so this is easier to copy-paste:

```suggestion
```sh
# Creates the migrations file.
python manage.py makemigrations
# Executes the migrations and updates the database with your model changes.
python manage.py migrate
```